assets/javascripts/admin_user.js

```diff
@@ -37,4 +37,25 @@ function setup_admin_user() {
             }
         });
     });
+
+    window.deleteUser = function(id){
+        var r = confirm("Are you sure you want to procede deleting this user?");
+        if (r == true) {
+            $.ajax({
+                url: '/api/v1/user/' + id,
+                method: 'DELETE',
+                dataType: 'json',
+                success: function() {
+                  addFlash('info', 'The user was deleted successfully. The page will be auto reloaded automatically');
+                  setTimeout(function(){
+                      location.reload();
```

Comment:
Do we really need to reload the entire page? Likely it is sufficient to simply remove the row from the DataTable (which has an API for that).

Reply:
Yes, this is an option. I'll do